### PR TITLE
[8.16] Update WMI event fields and add missing custom documentation fields

### PR DESCRIPTION
**Pull request by AsuNa-jp**
## Change Summary

* Reflected the feedbacks from @jdu2600 on the previous [PR](https://github.com/elastic/endpoint-package/pull/527).
* Added `process.Ext.api.parameters.operation` field (which I forgot to include it in the previous PR)
* Added the custom documentation for the newly added file event fileds in [this PR](https://github.com/elastic/endpoint-package/pull/514).
* Added the custom documentation for Win32k APIs and WMI APIs
  *  [windows_api_win32k.md](https://github.com/elastic/endpoint-package/pull/546/files#diff-ceb29ea53117aa2deddffc38ab5b1420e53d39ffc58864a72855816f422c0f98)
  * [windows_api_wmi.md](https://github.com/elastic/endpoint-package/pull/546/files#diff-8f0ee10d66d2298cd1a7db6935f96de2c0a66e45904eecd8dedcb52262240cd5)

## Sample values

Below are examples of the custom API fields (`process.Ext.api.parameters.operation`) that will be added by this PR. The sample of the document is here [sample_event.json](https://github.com/elastic/endpoint-package/blob/f5ea10c85738c6d942ed8bc80580d710c8a073a9/package/endpoint/data_stream/api/sample_event.json)

```json
    "message": "Malicious Behavior Detection Alert: EAF Rule",
    "process": {
        "Ext": {
            "api": {
                "metadata": {
                    "client_is_local": true
                },
                "name": "IWbemServices::CreateInstanceEnum",
                "parameters": {
                    "namespace": "root\\Microsoft\\Windows\\DeviceGuard",
                    "operation": "Win32_DeviceGuard"
                }
            },
```

## Release Target

8.16

### For mapping changes:

- [x] I ran `make` after making the schema changes, and committed all changes

### CODE REVIEW

**Review by jdu2600 — APPROVED** [review]
LGTM.
You'll need to resolve the conflicts due to my merge of https://github.com/elastic/endpoint-package/pull/545 and regenerate.

**Comment by AsuNa-jp** [comment]
Resolved the conflict between https://github.com/elastic/endpoint-package/pull/545.
So I think it is now ready for review! @elastic/security-defend-workflows

**Review by pzl — APPROVED** [review]

**Comment by AsuNa-jp** [comment]
Thanks for the review!

**Review by intxgo — APPROVED** [review]

File: package/endpoint/data_stream/api/fields/fields.yml [hunk 1]
```diff
@@ -1272,7 +1272,7 @@
...
...emory is unexpectedly fluctuating\n  \"hook_api\" - a change to the memory protection of a small executable image memory region was made\n  \"hollow_image\" - a change to the memory protection of a large executable image memory region was made\n  \"hook_unbacked\" - a change to the memory protection of a small executable non-image memory was made\n  \"hollow_unbacked\" - a change to the memory protection of a large executable non-image memory was made\n  \"guarded_code\" - executable memory was unexpectedly marked as PAGE_GUARD\n  \"hidden_code\" - executable memory was unexpectedly marked as PAGE_NOACCESS\n  \"execute_shellcode\" - a region of non-image executable memory was unexpectedly transferred control\n  \"hardware_breakpoint_set\" - a hardware breakpoint was set\n  \"rapid_background_polling\" - a suspicious process which does rapid input polling via GetAsyncKeyState API was observed\n  \"multiple_polling_processes\" - multiple suspicious processes which do rapid input polling via the GetAsyncKeyState API were observed\n  \"pid_spoofing\" - The acting process details may have been spoofed to hide the true origin\n  \"legacy_api\" - a deprecated or superseded API was called"
```
[the remark this answers is not in the document]
Comment (AsuNa-jp) [reply package/endpoint/data_stream/api/fields/fields.yml:1275]: You’re right, the wording is indeed a bit odd. I’ll check with John, but for now, I’ll go ahead and merge this PR as it is!

**Pull request merged**